docs/vsphere-golden-image.md

```diff
@@ -15,7 +15,8 @@ the patch is not available.*
 ## 2. Create the virtual machine
 
 To start with, create a new virtual machine in vSphere from the selected Windows Server distribution using the ISO image.
-See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/index.html) for installation guidelines.
+See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/index.html) for installation guidelines. We 
+recommend using a [virtual hardware version](https://kb.vmware.com/s/article/1003746) of `15` when creating the VM.
```

Comment:
```suggestion
See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/index.html) for installation guidelines. The supported [virtual hardware version](https://kb.vmware.com/s/article/1003746) is `15`.
```

Reply:
Re-reading this, we don't need to add another maintenance point WRT the hardware version.

For client-facing docs we should NOT set any explicit version, just a reference to [VMware vSphere infrastructure requirements](https://docs.openshift.com/container-platform/4.9/installing/installing_vsphere/preparing-to-install-on-vsphere.html#installation-vsphere-infrastructure_preparing-to-install-on-vsphere) for OCP will suffice, it reads: